src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -52,7 +52,9 @@ static void setCommonAttributes(SpanBuilder builder, RoutingContext routingConte
         builder.setAttribute(SemanticAttributes.HTTP_METHOD, routingContext.request().method().name()); // TODO remove in release after 0.28
         builder.setAttribute(SemanticAttributes.HTTP_REQUEST_METHOD, routingContext.request().method().name());
         builder.setAttribute(SemanticAttributes.HTTP_URL, routingContext.request().uri()); // TODO remove in release after 0.28
-        builder.setAttribute(SemanticAttributes.URL_FULL, routingContext.request().uri());
```

Comment:
`url.full` is a client specified attribute the [server side specification](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-server-semantic-conventions) breaks it up into component parts. However `url.full` is the documented replacement for `http.url`. So I'm taking the opportunity of deprecating `http.url` to improve the spec compliance.